LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/Region.cs

```diff
@@ -399,5 +260,17 @@ public int GetDRFromFreqAndChan(string datr)
         {
             return (int)this.DRtoConfiguration.FirstOrDefault(x => x.Value.configuration == datr).Key;
         }
+
+        public bool IsValidRX1DROffset(uint rx1DrOffset) => rx1DrOffset >= 0 && rx1DrOffset <= this.RX1DROffsetTable.GetUpperBound(1);
+
+        public bool IsValidRXDelay(ushort desiredRXDelay)
+        {
+                if (desiredRXDelay >= 0 && desiredRXDelay <= MAX_RX_DELAY)

```

Comment:
Indent is wrong.
Can be changed to 
return desiredRXDelay >= 0 && desiredRXDelay <= MAX_RX_DELAY;